src/subgraph/zns/actions/getMostRecentSubdomainsById.ts

```diff
@@ -16,12 +16,18 @@ export const getMostRecentSubdomainsById = async <T>(
 ): Promise<Domain[]> => {
   const subDomains: Domain[] = [];
   let yetUnreceived = count;
+  let queriedDomains: DomainDto[] = [];
   if (count >= MAX_RECORDS) {
     throw new Error(
       `Please request no more than ${MAX_RECORDS} records at a time.`
     );
   }
-  while (true) {
+  /**
+   * We will only get back up to `queryCount` # of domains
+   * So if we get that many there's probably more domains we need
+   * to fetch. If we got back less, we can stop querying
+   */
+  do {
```

Comment:
Many of these loops had to be altered to pass the linting step.
I made them into do whiles, because it was most similar to the structure they were already using